utils/oscap-xccdf.c

```diff
@@ -1041,6 +1046,9 @@ bool getopt_xccdf(int argc, char **argv, struct oscap_action *action)
                 } else {
                     action->f_ovals = NULL;
                 }
+
+		if (action->rule != NULL && action->profile == NULL)
+			return oscap_module_usage(action->module, stderr, "Profile needs to be specified when evaluating a single rule!");
```

Comment:
@jan-cerny Thanks for explanation, I removed this "if" chunk of code and I agree that printing error does not make sense if profile is NULL.